src/pmod_maxsonar.erl

```diff
@@ -45,9 +45,13 @@ handle_cast(Request, _State) -> error({unknown_cast, Request}).
 % @private
 handle_info({Port, {data, Data}}, #state{port = Port} = State) ->
     case Data of
-        <<_, _, D1, D2, D3, 10>> when $0 =< D1, D1 =< $9,
+        % Format of response is 'Rxxx\n' where xxx is the decimal
+        % representation of the measured range in inches (2.54cm)
+        % (left-padded with zeros - so there are always three digits)
+        <<82, D1, D2, D3, 10>> when $0 =< D1, D1 =< $9,
```

Comment:
Could you change this to `<<$R, D1, D2, D3, $\n>>` instead? Matches better with the comment 😄

Reply:
Done.
(Learned something new about erlang :blush:)